Deployment/Configure-Alarms.ps1

```diff
@@ -0,0 +1,58 @@
+<#
```

Comment:
Thinking of these scripts. It could be good idea to create re-usable module that simplifies common use case we often use. Even on level like "Set-SqlServerMonitoring ...", "Set-WebAppMonitoring" and publish that module on psget so it's easy to share.

However that common module is definetly out of scope of this PR, lets see if we can find enough common ground between projects to reason that module.

Reply:
I think that sounds useful.